<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formatwindow</class>
 <widget class="QWidget" name="formatwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/tu/Res/btn_nosel.png);</string>
  </property>
  <widget class="QLabel" name="m_pLb_Msg">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>901</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size: 55px;font: bold;background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Confirm">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_confirm.png);
QPushButton{color:white;border:none;background:transparent;font-size: 35px;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Cancel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>540</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_cancel.png);
color:white;border:none;background:transparent;font-size: 35px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
